### PR TITLE
Dependencies updates for the week of May 02, 2022

### PR DESCRIPTION
**Pull request by rgoshen**
Updated Jest from version 27 to 28.  Manually added jest to babel calls by adding rootMode: upward.  This resolved babel failure to transpile ECMA vs CommonJS package imports. Next, created jest-browser.confing.js file. This resolved throwing of TextEncoder not defined issue.  Also, in the jest-browser.config.js, added moduelNameMapper for known uuid entry point for browser vs. node issue.

@jimlerza this will require a new docker image to be pushed to ustaxcourt aws environments.

### CODE REVIEW

File: web-client/JsdomWithTextEncoderEnvironment.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+// const JsdomEnvironment = require('jest-environment-jsdom');
```
Comment (rachelschneiderman) [inline web-client/JsdomWithTextEncoderEnvironment.js:1]: Nit: can this be removed?

Comment (rosiehsmith) [inline web-client/JsdomWithTextEncoderEnvironment.js]: ⛏️  leftover comment!

File: package.json [hunk 2]
```diff
@@ -269,43 +270,44 @@
...
-    "eslint": "^8.13.0",
     "esm": "^3.2.25",
     "file-loader": "^6.2.0",
     "html-webpack-plugin": "^5.5.0",
     "html-webpack-pug-plugin": "^4.0.0",
     "husky": "^7.0.4",
-    "jest": "^27.5.1",
+    "jest": "^28.0.3",
+    "jest-environment-jsdom": "^28.0.2",
+    "jest-environment-node": "^28.0.2",
```
Comment (rosiehsmith) [inline package.json:280]: nice handling of this major update! 🎉

File: docs/team-process.md [hunk 3]
```diff
@@ -70,13 +70,15 @@ If dependencies have no patch, replace it with an alternative, or wait for the l
...
 
     - Once verification is complete, you will need to increment the docker image version being used in `.circleci/config.yml` and publish a docker image tagged with the incremented version number to ECR for both Flexion and USTC accounts.
 
-4. `docker`: Update [docker base image](https://hub.docker.com/r/cypress/base/tags?page=1&name=14.) if an update is available for the current node version the project is using.
+5. `docker`: Update [docker base image](https://hub.docker.com/r/cypress/base/tags?page=1&name=14.) if an update is available for the current node version the project is using.
 
-5. (Optional) Check through the list of caveats to see if any of the documented issues have been resolved.
+    See [here](ci-cd.md#docker) for the documentation to create and push the updated docker container for use in CircleCI.
+
+6. (Optional) Check through the list of caveats to see if any of the documented issues have been resolved.
```
Comment (rosiehsmith) [inline docs/team-process.md:82]: I didn't see in the exp runs if y'all did a CircleCI build with a forced migration. Am I missing that? Seeing that it's way below this section in `Validating Updates` I know it's not too apparent that it's part of the update steps.

Comment (halprin) [reply docs/team-process.md:82]: Ooooh, good catch.  We didn't do that.

File: web-client/JsdomWithTextEncoderEnvironment.js (hunk 1 above)
Comment (halprin) [reply web-client/JsdomWithTextEncoderEnvironment.js]: Fixed!

**Review by rosiehsmith — APPROVED** [review]

**Review by rachelschneiderman — APPROVED** [review]

**Review by mmarcotte — APPROVED** [review]

**Pull request merged**